src/ucm/Makefile.am

```diff
@@ -35,7 +35,8 @@ noinst_HEADERS = \
 	util/replace.h \
 	util/log.h \
 	util/reloc.h \
-	util/sys.h
+	util/sys.h \
+	util/bistro.h
```

Comment:
maybe add a subdirectory for "binary patching" and have a separate file for every architecture?
e.g not put it in "util"